services/user-tenant-service/src/test/java/com/sourcefuse/jarc/services/usertenantservice/mocks/MockCurrentUserSession.java

```diff
@@ -63,4 +67,14 @@ public static CurrentUser getCurrentUser() {
       .getAuthentication()
       .getPrincipal();
   }
+
+  public static Specification getSpecification(QueryService queryService) {
```

Comment:
keep this mock seperate as this file is related to current user session

Reply:
done